streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -137,6 +138,7 @@ public StreamTask(final TaskId id,
         processRatioSensor = TaskMetrics.activeProcessRatioSensor(threadId, taskId, streamsMetrics);
         processLatencySensor = TaskMetrics.processLatencySensor(threadId, taskId, streamsMetrics);
         punctuateLatencySensor = TaskMetrics.punctuateSensor(threadId, taskId, streamsMetrics);
+        bufferedRecordsSensor = TaskMetrics.activeBufferedRecordsSensor(threadId, taskId, streamsMetrics);
```

Comment:
req: Please verify the creation of those metrics in `StreamTaskTest#shouldConstructMetricsWithBuiltInMetricsVersionLatest()`.

Reply:
Ack.